assets/interfaces.go

```diff
@@ -287,6 +287,19 @@ type Ticketer interface {
 	Type() string
 }
 
+// User is an person who can trigger flows or be assigned tickets etc.
+//
+//   {
+//     "email": "bob@nyaruka.com",
+//     "name": "Bob"
+//   }
+//
+// @asset user
+type User interface {
+	Email() string
+	Name() string
+}
```

Comment:
I've made users be assets even tho we don't really need them to be just yet because it seems inevitable that flows will need to know about users at some point - users will want to do ticket assignment in the actual flow action etc